src/types/aggregation.assert.ts

```diff
@@ -0,0 +1,53 @@
+import * as ta from 'type-assertions'
+import { Pipeline } from './aggregation'
+import type { Collstats } from './colstats'
+import { Coordinates, GeoNear } from './geoNear'
+import { Limit } from './limit'
+import { Skip } from './skip'
+
+type CorrectExample = number
+type IncorrectExample = string
+
+type CorrectCollStatsExample = {
+    latencyStats: { histograms: boolean },
+    storageStats: { scale: number },
+    count: {},
+    queryExecStats: {}
+}
+
+type IncorrectCollStatsExample = {
+    latencyStats: { histograms: number },
+    storageStats: { scale: string },
+    count: {},
+    queryExecStats: {}
+}
```

Comment:
You should test each thing being wrong separately, not all of them being wrong at the same time.